### PR TITLE
new package: bun

### PR DESCRIPTION
**Pull request by jothi-prasath**
#571

### CODE REVIEW

**Comment by licy183** [comment]
Not working. See https://github.com/termux-user-repository/tur/issues/571#issuecomment-1721686680.

File: tur/bun/build.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://bun.sh/
+_MAJOR_VERSION=1.0
```
Comment (asumbek) [inline tur/bun/build.sh:2]: There's no need for this, just `TERMUX_PKG_VERSION=1.2`.

**Review by UrNightmaree — COMMENTED** [review]
This package is literally a clownshow. Why the hell it grabs prebuilt binary from oven-sh/bun, especially `linux-aarch64` prebuilt? Termux isn't Linux, so Linux binaries wouldn't work inside Termux without QEMU or proot.

**Comment by jothi-prasath** [comment]
@UrNightmaree it can run without proot or qemu

https://github.com/termux-user-repository/tur/issues/571#issuecomment-1721686680
https://github.com/termux/termux-packages/issues/11188

**Comment by Maxython** [comment]
@jothi-prasath, you can create a request to add the `bun` package to the [termux-pacman/glibc-packages](https://github.com/termux-pacman/glibc-packages/issues/new?assignees=&labels=package+request%2Cgpkg&projects=&template=package_request.yml&title=%5BPackage%5D%3A+) repository. It will be added to the `gpkg` repo.

**Pull request closed without merging**